README.md

```diff
@@ -55,19 +56,21 @@ It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controlle
 which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster 
 
 ### Test It Out
-1. Install the CRDs into the cluster:
-
+* Install the CRDs and the operator-controller into a new [KIND cluster](https://kind.sigs.k8s.io/):
 ```sh
-make install
+make run
 ```
+This will build a local container image of the operator-controller, create a KIND cluster and then install onto that KIND cluster.
+This will also deploy the catalogd, rukpak and cert-manager dependencies.
 
-2. Run your controller (this will run in the foreground, so switch to a new terminal if you want to leave it running):
+**OR**
 
+* Create deplpoyment files
```

Comment:
Typo:
```suggestion
* Create deployment files
```

Reply:
I have a big problem with extra `p`s in deployment, apparently.